circleci-provision-scripts/base.sh

```diff
@@ -45,9 +45,12 @@ EOF
     export DEBIAN_FRONTEND=noninteractive
 
     apt-get update -y
+    apt-get install -y software-properties-common
+    apt-get update -y
+    apt-add-repository ppa:git-core/ppa
 
     # Install base packages
-    cat << EOS | xargs apt-get install
+    apt-get install $(tr "\n" " " <<EOS
```

Comment:
Can you explain what you mean?

I don't think we should quote the subexpression since `apt-get install "a b"` is different from `apt-get install "a" "b"`.

I should use single-quotes on the args to `tr` though, one sec.